contrib/packaging_python/mk_wheel.py

```diff
@@ -0,0 +1,118 @@
+###############################################################################
+# Top contributors (to current version):
+#   Makai Mann, Gereon Kremer
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# ############################################################################
+#
+# Build script for python wheels for cvc5.
+# It provides two main functions:
+# - get_cvc5_version() uses the version.cmake script to obtain the proper cvc5
+#   version string. It is extracted from the versioninfo.cpp file and passed on
+#   to the setuptools config.
+# - build_extension() builds the cvc5 python api(s) using the cvc5_python_api
+#   target and copying it to wherever setuptools expects it to be.
+#
+# Example usage (from build directory):
+#   python3 ../contrib/packaging_python/mk_wheel.py bdist_wheel -d dist
+##
+
+import os
+import re
+import platform
+import subprocess
+import shutil
+
+from setuptools import setup, Extension
+from setuptools.command.build_ext import build_ext
+from distutils.version import LooseVersion
+
+
+def get_project_src_path():
+    # expecting this script to be in contrib/packaging_python/
+    # The project source directory is two directories up
+    name = __file__
+    for i in range(3):
+        name = os.path.dirname(name)
```

Comment:
Can we maybe add checks to ensure that the directory returned is indeed the project source directory? E.g. check if the directory returned contains directory `src` or something like that?

Reply:
Hmm, we also have a `src` directory in our build folders. Maybe checking for `configure.sh` would be a bit safer?